### DISCUSSION TITLE
Member Meeting Minutes 2024-5-15

### DISCUSSION THREAD

**Discussion by node** [PECommunity/community#51] [General]
### 会议出席人员
请在评论区 +1 

### 本期会议讨论议题
- 网站首页： 平台工程基本概念、关键特征、倡导的关键行动、相关项目与框架 概述，提升新手的学习效率。 
- 急缺原创内容：Call for action 
- 双周会议优化：一次社区会议 + 一次公开的在线平台工程讨论会  （参考 Platform Coffee）

请继续补充你提出的议题。

**Comment by Jeremy-boo** [comment]
+1

**Comment by cr7258** [comment]
+1

**Comment by MarcelusXie** [comment]
+1

**Comment by node** [comment]
国内外相关公开 Showcase 整理呈现至首页。

**Comment by node** [comment]
讨论会：
1. 如果有1个分享，可以围绕分享主题讨论；或者2～4个人就某个主题讨论，其他人参与；
1. 时间：人太多，时间可能不太可控
2. 参与度：参与讨论的人数 VS 听众的人数 
1. 明确的讨论主题；提前准备“预习”材料

**Comment by node** [comment]
讨论内容：
- 科普/布道类的 
- 分享&共识更加适合本土的实践方法，不只是停留在搬运国外的阶段
- 找到并分享一些特定场景的平台工程实施的沙盘模拟/推演 ：如售前、向上汇报、国外案例在本土企业内部的参考意义&集中讨论 等 
- DevOps相关厂商的参与，对平台工程的认知&实践，度量相关的平台工程价值

**Comment by node** [comment]
社区协作形式：
- 例如 IDPBuilder ，这类项目可以组团做一些探索和DEMO

**Comment by yimeng** [comment]
+1
